aggregate_test.go

```diff
@@ -26,6 +26,46 @@ func TestAggregate(t *testing.T) {
 	}
 }
 
+func TestAggregateT(t *testing.T) {
```

Comment:
When I wrote this tests, my concerns was just code coverage. I didn't realize that the examples tests are included in code coverage metrics. Nice catch!